src/main.rs

```diff
@@ -1,15 +1,68 @@
 use clap::{crate_authors, crate_description, crate_version, Clap};
+mod bangumi;
```

Comment:
see https://stackoverflow.com/questions/26946646/rust-package-with-both-a-library-and-a-binary/26953326
use crate::bangumi ?

Reply:
It's no necessary to be a library, I guess...

Reply:
`use dantalian::bangumi;`
should be like this(